data/other.yml

```diff
@@ -0,0 +1,61 @@
+---
+- name: Awesome Newsletters Ruby
+  url: https://github.com/zudochkin/awesome-newsletters#ruby
+  rss:
+  locked: true
+- name: Planet Argon - 2020 Ruby on Rails Community Survey Results
+  url: https://rails-hosting.com/2020/#learning
+  rss:
+  locked: true
+- name: markets/awesome-ruby
+  url: https://github.com/markets/awesome-ruby
+  rss: https://github.com/kigster/simple-feed
+  locked: true
+- name: sdogruyol/awesome-ruby
+  url: https://github.com/sdogruyol/awesome-ruby
+  rss:
+  locked: true
+- name: gramantin/awesome-rails
+  url: https://github.com/gramantin/awesome-rails
+  rss: https://github.com/davidesantangelo/api.rss
```

Comment:
@AlexB52 We need to somehow handle such cases

Reply:
I'm not sure how to handle this other than ignoring them like we do for web archive links. 

Otherwise, what do you think of manually updating these links and locking them down? I can remove the links or manually look for them. 

If one of these gets updated again, in the future with a force :all or :norss update we might be able to spot the change in the diffs easily and remove them.